chains/v3/chains_dev.json

```diff
@@ -2114,7 +2114,7 @@
         ],
         "nodes": [
             {
-                "url": "wss://mandala.polkawallet.io",
+                "url": "wss://wss://rpc.pinknode.io",
```

Comment:
double `wss://`

Reply:
wss://rpc.pinknode.io/mandala

Reply:
or `wss://acala-mandala.api.onfinality.io/public-ws`

Reply:
Thank you, fixed